pkg/idaes_examples/common/hda/hda_ideal_VLE.py

```diff
@@ -376,44 +376,35 @@ def build(self):
     def define_metadata(cls, obj):
         """Define properties supported and units."""
         obj.add_properties(
-            {'flow_mol': {'method': None, 'units': 'mol/s'},
-             'flow_mol_phase_comp': {'method': None, 'units': 'mol/s'},
-             'mole_frac_comp': {'method': None, 'units': 'none'},
-             'temperature': {'method': None, 'units': 'K'},
-             'pressure': {'method': None, 'units': 'Pa'},
-             'flow_mol_phase': {'method': None, 'units': 'mol/s'},
-             'dens_mol_phase': {'method': '_dens_mol_phase',
-                                'units': 'mol/m^3'},
-             'pressure_sat': {'method': '_pressure_sat', 'units': 'Pa'},
-             'mole_frac_phase_comp': {'method': '_mole_frac_phase',
-                                      'units': 'no unit'},
+            {'flow_mol': {'method': None},
+             'flow_mol_phase_comp': {'method': None},
+             'mole_frac_comp': {'method': None},
+             'temperature': {'method': None},
+             'pressure': {'method': None},
+             'flow_mol_phase': {'method': None},
+             'dens_mol_phase': {'method': '_dens_mol_phase'},
+             'pressure_sat': {'method': '_pressure_sat'},
+             'mole_frac_phase_comp': {'method': '_mole_frac_phase'},
              'energy_internal_mol_phase_comp': {
-                     'method': '_energy_internal_mol_phase_comp',
-                     'units': 'J/mol'},
+                     'method': '_energy_internal_mol_phase_comp'},
              'energy_internal_mol_phase': {
-                     'method': '_enenrgy_internal_mol_phase',
-                     'units': 'J/mol'},
-             'enth_mol_phase_comp': {'method': '_enth_mol_phase_comp',
-                                     'units': 'J/mol'},
-             'enth_mol_phase': {'method': '_enth_mol_phase',
-                                'units': 'J/mol'},
-             'entr_mol_phase_comp': {'method': '_entr_mol_phase_comp',
-                                     'units': 'J/mol'},
-             'entr_mol_phase': {'method': '_entr_mol_phase',
-                                'units': 'J/mol'},
-             'temperature_bubble': {'method': '_temperature_bubble',
-                                    'units': 'K'},
-             'temperature_dew': {'method': '_temperature_dew',
-                                 'units': 'K'},
-             'pressure_bubble': {'method': '_pressure_bubble',
-                                 'units': 'Pa'},
-             'pressure_dew': {'method': '_pressure_dew',
-                              'units': 'Pa'},
-             'fug_vap': {'method': '_fug_vap', 'units': 'Pa'},
-             'fug_liq': {'method': '_fug_liq', 'units': 'Pa'},
-             'dh_vap': {'method': '_dh_vap', 'units': 'J/mol'},
-             'ds_vap': {'method': '_ds_vap', 'units': 'J/mol.K'}})
-
+                     'method': '_enenrgy_internal_mol_phase'},
```

Comment:
Typo "enenrgy". I followed the commit history and this typo has been here since the original file was added, which then brings up whether there should have been a warning or error that indicated an incorrect method name.

Reply:
@bpaul it would be possible to verify the names, but we generally use the approach and "try it and let it fail then". In this case, it just means this property was never used by an example.